config/initializers/as_to_time.rb

```diff
@@ -14,4 +14,22 @@ def to_time(form = OBJ)
       old_to_time(form)
     end
   end
+
+  IEC_SIZE_SUFFIXES = %w(Ki Mi Gi Ti).freeze
+  def parse_iec_number()
```

Comment:
I would be very careful to extend core classes.

In case you go down that route
* name the method `to_iec_integer` to be in line with other conversion methods
* add another initializer and dont hijack `as_to_time.rb` 
* or add it to `gems/pending/util/extensions` which is probably a better place
* add a link to the iec standard this should implement

@Fryguy ?